hugo/content/grafana/_index.md

```diff
@@ -11,6 +11,7 @@ There are RPM packages available to [Crunchy Data](https://www.crunchydata.com)
 - [Upgrading](#upgrading)
 - [Setup](#setup)
     - [Linux](#setup-on-linux)
+- [Included Dashboards](#dashboards)
 
 ## Installation {#installation}
```

Comment:
Move alertmanager section out of prometheus ?
http://localhost:1313/#installation
1. exporter
2. Prometheus
3. Alertmanager
4. Grafana
My 2 cents - with alertmanager HA it like a product

Reply:
Made an issue to see if we can expand alertmanager docs and separate them out to their own 

https://github.com/CrunchyData/pgmonitor/issues/271